Doc/library/concurrent.futures.rst

```diff
@@ -173,8 +173,8 @@ And::
 
    .. versionchanged:: 3.8
       Default value of *max_workers* is changed to ``min(32, os.cpu_count() + 4)``.
-      This default value preserves at least 5 workers for I/O bound tasks.
-      It utilizes at most 32 CPU cores for CPU bound tasks which release the GIL.
+      This default value preserves at least 5 workers for I/O-bound tasks.
+      It utilizes at most 32 CPU cores for CPU-bound tasks which release the GIL.
```

Comment:
In both of these cases, `I/O bound` and `CPU bound` precede `tasks`, so they should be hyphenated.

https://www.grammarly.com/blog/hyphen/